### DISCUSSION TITLE
What is the equivalent of Array field? Best field for array type.

### DISCUSSION THREAD

**Discussion by weidongguo** [AntlerVC/firetable#412] [Support / Q&A] [answered]
HOW DO WE ENTER ARRAY?

The closest I can find it's Multi-select, which only takes strings of predefined options. It hints me that the user can enter custom values, but so far, I DON'T how to turn that option on.

How about array of integers?

Alternatively, I try JSON, but it doesn't seem like to allow array at the top level even when I turn on "Set as an Array" IN THE COLUMN SETTINGS.

**Comment by weidongguo** [comment]
@shamsmosowi any recommendations?

**Comment by harinij** [comment, marked as answer]
Late here, but this is answered here if still useful https://github.com/rowyio/rowy/discussions/731
